### PR TITLE
Fix confirm screen Avatar href

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#575** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/575?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/utils/useProfileHref.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { baseMainnet } from '@my/wagmi'
+import { useProfileLookup } from './useProfileLookup'
+
+export const useProfileHref = (
+  lookup_type: Database['public']['Enums']['lookup_type_enum'],
+  identifier: string
+) => {
+  // refcode and phone cannot be linked directly to. This will disable the query when its not needed
+  const isEnabled = ['refcode', 'phone'].includes(lookup_type)
+  const { data: profile, error } = useProfileLookup(lookup_type, identifier ?? '', isEnabled)
```
Comment (0xBigBoss) [inline packages/app/utils/useProfileHref.ts:11]: seems really hacky to include the data fetching in this component. IMO we should separate the profile href and block explorer. The calling component will have to do this, or let's update this hook to accept more args and cascading logic.

1. profile.tag
2. profile.sendid
3. basescan address

Comment (youngkidwarrior) [reply packages/app/utils/useProfileHref.ts:11]: Yeah this is doing too much. I'll separate out the logic

Comment (youngkidwarrior) [reply packages/app/utils/useProfileHref.ts]: I'm actually not quite sure what direction you want on this actually.

I just took out the profile lookup for now. Let me know what you think

**Pull request merged**